nbs/tsdataset.ipynb

```diff
@@ -198,36 +198,35 @@
     "            return False\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.        def update_dataset(dataset, future_df):
</pre><p>Is there a way to test this method?</p><p> </p><p>Additionally, perhaps we could standardize the name of the future df argument,&nbsp;<code>futr_df</code>&nbsp;vs&nbsp;<code>future_df</code> in the <code>NeuralForecast.predict</code> method.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
we are alredy testing it